modules/common.nix

```diff
@@ -5,81 +5,125 @@ let
+    };
+  };
+  config = {
+    networking.hostName = name;
 
-  networking.hostName = name;
+    environment.systemPackages = with pkgs; [
+      (ruby.withPackages (ps: with ps; [ sequel pry sqlite3 nokogiri ]))
+      bat
+      di
+      fd
+      file
+      git
+      graphviz
+      htop
+      iptables
+      jq
+      lsof
+      mosh
+      ncdu
+      ripgrep
+      screen
+      sqlite-interactive
+      sysstat
+      tcpdump
+      tig
+      tmux
+      tree
+    ] ++ (lib.optional config.local.commonGivesVim vim);
 
-  environment.systemPackages = with pkgs; [
-    bat
-    git
-    graphviz
-    htop
-    iptables
-    jq
-    lsof
-    mosh
-    ncdu
-    sysstat
-    tcpdump
-    tig
-    tree
-    vim
-  ];
+    environment.variables.TERM = "xterm-256color";
 
-  environment.variables.TERM = "xterm-256color";
+    users.mutableUsers = false;
+    users.users.root.openssh.authorizedKeys.keys = devOpsKeys;
 
-  users.mutableUsers = false;
-  users.users.root.openssh.authorizedKeys.keys = devOpsKeys;
+    services = {
+      monitoring-exporters.graylogHost =
+        if config.deployment.targetEnv == "ec2"
+        then "monitoring-ip:5044"
+        else "monitoring:5044";
 
-  services = {
-    monitoring-exporters.graylogHost =
-      if config.deployment.targetEnv == "ec2"
-      then "monitoring-ip:5044"
-      else "monitoring:5044";
+      openssh = {
+        passwordAuthentication = false;
+        authorizedKeysFiles = lib.mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
+        extraConfig = lib.mkOrder 9999 ''
+          Match User root
+            AuthorizedKeysFile .ssh/authorized_keys .ssh/authorized_keys2 /etc/ssh/authorized_keys.d/%u
+        '';
+      };
 
-    openssh = {
-      passwordAuthentication = false;
-      authorizedKeysFiles = lib.mkForce [ "/etc/ssh/authorized_keys.d/%u" ];
-      extraConfig = lib.mkOrder 9999 ''
-        Match User root
-          AuthorizedKeysFile .ssh/authorized_keys .ssh/authorized_keys2 /etc/ssh/authorized_keys.d/%u
-      '';
+      ntp.enable = true;
+      cron.enable = true;
     };
 
-    ntp.enable = true;
-    cron.enable = true;
-  };
+    nix = rec {
+      # use nix sandboxing for greater determinism
+      useSandbox = true;
 
-  nix = rec {
-    # use nix sandboxing for greater determinism
-    useSandbox = true;
+      # make sure we have enough build users
+      nrBuildUsers = 32;
 
-    # make sure we have enough build users
-    nrBuildUsers = 32;
+      # if our hydra is down, don't wait forever
+      extraOptions = ''
+        connect-timeout = 10
+        http2 = true
+        show-trace = true
+        narinfo-cache-negative-ttl = 0
+      '';
 
-    # if our hydra is down, don't wait forever
-    extraOptions = ''
-      connect-timeout = 10
-      http2 = true
-      show-trace = true
-    '';
+      # use all cores
+      buildCores = 0;
 
-    # use all cores
-    buildCores = 0;
+      #nixPath = [ "nixpkgs=/run/current-system/nixpkgs" ];
```

Comment:
should we comment why this is commented? (btw, why was it needed before and not anymore?)